src/app/planner/card/card.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="card" >
+		<div class="card-info">
+		<img class="meal-img" src='{{url}}' alt="meal">
```

Comment:
For source it is better to use the other property binding way [src]="url". 

{{}} is more when you want to directly display some text in your template